core-web/apps/dotcms-ui/src/app/portlets/dot-edit-page/seo/components/dot-edit-page-state-controller-seo/dot-edit-page-state-controller-seo.component.ts

```diff
@@ -0,0 +1,302 @@
+import { from, Observable, of } from 'rxjs';
```

Comment:
Are you 100% sure you have to dupe all this component? I don't have the UI in my head but worth to check.